code/modules/jobs/job_types/shaft_miner.dm

```diff
@@ -28,9 +28,9 @@
 	gloves = /obj/item/clothing/gloves/color/black
 	uniform = /obj/item/clothing/under/rank/miner/lavaland
 	l_pocket = /obj/item/reagent_containers/hypospray/medipen/survival
-	r_pocket = /obj/item/flashlight/seclite
+	r_pocket = /obj/item/storage/bag/ore	//causes issues if spawned in backpack
```

Comment:
can you explain?

Reply:
Sure. The bag relies on a mobhook component which is pre-filled when it's spawned in bag. I didn't bother digging too deep, but I assume when you spawn in with gear, /equipped() is called. If you just leave the bag in your backpack, the bag behaves as if it is in your belt/pocket slot - regardless of where the backpack is.
Spawning it in your pocket/belt instead forces you to call /dropped() on it if you put it somewhere else, deleting the mobhook.

Reply:
cringe